dashboard/app/views/levels/_dialogHelper.js

```diff
@@ -143,5 +143,11 @@
       }
     });
   }
+
+  // Export the functions which are provided by this file.
+  dashboard.dialog = {
+    showStartOverDialog: showStartOverDialog,
+    processResults: processResults
+  };
 })();
```

Comment:
I might make this window.dashboard (just to make it explicit that dashboard is a global).

Also might want to first do window.dashboard = window.dashboard || {};
in case you're the first to define dashboard.